infra/scripts/node.toml

```diff
@@ -1,19 +1,36 @@
 ########################################################################################################################
 # Node helpers
 ########################################################################################################################
+
+[tasks.bootstrap]
+env = { "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_DIR}/bootstrap/cometbft", "CMT_CONTAINER_NAME" = "cometbft-${NETWORK_NAME}-bootstrap", "CMT_P2P_HOST_PORT" = "26650"}
```

Comment:
I might be misunderstanding cargo make, but [the docs](https://github.com/consensus-shipyard/fendermint/pull/320/files#diff-9265726f2e486a1f8a407d6041312142989b59d7d59960399fc4e7085e203125R35-R37) say we can define the bootstrap host port, but here it's hardcoded.

Reply:
Removed, it may be confusing. Let's leave users choose their port instead of forcing a different one for them.